tests/validator/test_validation_graph.py

```diff
@@ -38,14 +50,62 @@ def expect_column_values_to_be_unique_expectation_config() -> ExpectationConfigu
 
 
 @pytest.fixture
-def expectation_validation_graph(
+def expect_column_value_z_scores_to_be_less_than_expectation_config() -> ExpectationConfiguration:
+    return ExpectationConfiguration(
+        expectation_type="expect_column_value_z_scores_to_be_less_than",
+        kwargs={
+            "column": "a",
+            "mostly": 0.9,
+            "threshold": 4,
+            "double_sided": True,
+        },
+    )
+
+
+@pytest.fixture
+def expect_column_values_to_be_unique_expectation_validation_graph(
     expect_column_values_to_be_unique_expectation_config: ExpectationConfiguration,
 ) -> ExpectationValidationGraph:
     return ExpectationValidationGraph(
         configuration=expect_column_values_to_be_unique_expectation_config
     )
 
 
+@pytest.fixture
+def expect_column_value_z_scores_to_be_less_than_expectation_validation_graph(
+    expect_column_value_z_scores_to_be_less_than_expectation_config,
+):
+    class PandasExecutionEngineStub:
+        pass
+
+    PandasExecutionEngineStub.__name__ = "PandasExecutionEngine"
+    pandas_execution_engine_stub = cast(ExecutionEngine, PandasExecutionEngineStub())
```

Comment:
@donaldheppner @Kilo59 specifically did not want a fixture, until we all had a chance to discuss.  Can we put it up on the agenda soon?  Thank you.

Reply:
My request not to have it in a fixture was because I think the value of being to clearly see that the Dummy doesn't define any behavior (and is a true "dummy") outweighs the minor duplication.

But we almost certainly do want to use fixtures and remove duplication when the Double (Mock, Stub, Fake etc.) becomes more complex. Although I don't know where the line is there. I think we should decide this as a team.